extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -283,8 +283,13 @@ class AmpAccordion extends AMP.BaseElement {
     // overidden.
     const target = dev().assertElement(event.target);
     const header = dev().assertElement(event.currentTarget);
-    const anchor = closest(target, e => e.tagName == 'A', header);
-    if (anchor === null) {
+    const anchorOrTapTarget = closest(target, e => {
+      return (e.tagName == 'A')
+        || (e.hasAttribute('on')
+          && e.getAttribute('on')./*OK*/matches(/(^|;)\s*tap\s*/));
```

Comment:
Could you please create an issue to expose a 'hasTapAction(elem)` on `actions` service? It already parses the `on` attribute and does not need to rely on RegEx. With this change now at least three places that are doing a similar thing (STAMP, lightbox and this) so worth having a first-class support for it.

Reply:
with current implementation, I suggest at least caching the result of '.matches' in an instance variable so we don't repeat it on every click. (`on` attribute is static and can not be changed via `amp-bind`)

Reply:
That's a good idea! Will it suffice to expose a wrapper around [`findAction_(target, actionEventType)`](https://github.com/ampproject/amphtml/blob/master/src/service/action-impl.js#L524-L540)?

Reply:
So I gave this a try. I think caching is a bit weird here because we don't really have a clean unique identifier for elements, since the target can technically be any child of a header and it's not necessarily the case that elements will have ids. Is there a more intelligent way to uniquely identify these elements for caching? Or should caching happen at the `ActionService` level?